setup.cfg

```diff
@@ -34,7 +34,14 @@ universal = 1
 [flake8]
 max-line-length = 99
 exclude = docs, .tox, .git, __pycache__, .ipynb_checkpoints
-ignore = W503
+extend-ignore = W503, D107, SFS3, # String literal formatting using f-string.
+    VNE001  # Single letter variable names are not allowed
```

Comment:
Add the `pydocstyle` error codes that we want to capture here:
```
    D107,  # Missing docstring in __init__
    D417   # Missing argument descriptions in the docstring, this is a bug from pydocstyle: https://github.com/PyCQA/pydocstyle/issues/449
``` 
pd: no need of `D407`

Reply:
Also add `docstring-convetion = google`  to the `flake8`.